src/servers/src/grpc/greptime_handler.rs

```diff
@@ -118,6 +126,70 @@ impl GreptimeRequestHandler {
             None => result_future.await,
         }
     }
+
+    pub(crate) async fn put_record_batches(
+        &self,
+        mut stream: PutRecordBatchRequestStream,
+        result_sender: mpsc::Sender<TonicResult<GreptimeResponse>>,
+    ) {
+        // TODO(LFC): handle authentication, too
+
+        let handler = self.handler.clone();
+        let runtime = self
+            .runtime
+            .clone()
+            .unwrap_or_else(common_runtime::global_runtime);
+        runtime.spawn(async move {
+            let mut table_to_put: Option<TableName> = None;
+
+            while let Some(request) = stream.next().await {
+                let request = match request {
+                    Ok(request) => request,
+                    Err(e) => {
+                        let _ = result_sender.try_send(Err(e));
```

Comment:
Should we ignore the error returned by `try_send` when the mpsc channel is full?

Reply:
If `try_send` has any error, it means the resources in greptimedb side are very busy or the client lost connection. In either case the while loop will break, then the stream will be terminated, just what we want.